test/e2e/erc721/sameChain.js

```diff
@@ -1,126 +0,0 @@
-const TruffleAssert = require('truffle-assertions');
```

Comment:
Let's revert it. We don't use ERC721 but still have in the repo.

Reply:
Reverted the changes